rockspec/apisix-master-0.rockspec

```diff
@@ -70,7 +70,6 @@ dependencies = {
     "ext-plugin-proto = 0.6.1",
     "casbin = 1.41.8-1",
     "inspect == 3.1.1",
-    "lualdap = 1.2.6-1",
```

Comment:
@kingluo 

> Why still compile lualdap? [apache/apisix/actions/runs/6357262095/job/17274627068?pr=10176](https://github.com/apache/apisix/actions/runs/6357262095/job/17274627068?pr=10176)

`lualdap = 1.2.6` dependency is removed here, probably the CI tools does not reflect this change ?